Terminal.Gui/Views/TextField.cs

```diff
@@ -242,20 +241,26 @@ static int SetCol (int col, int width, int cols)
 			return col;
 		}
 
-		// Returns the size in a range of the string.
-		int DisplaySize (List<Rune> t, int start = -1, int end = -1)
+		// Returns the size and length in a range of the string.
+		(int, int) DisplaySize (List<Rune> t, int start = -1, int end = -1, bool checkNextRune = true)
```

Comment:
I agree!

Reply:
@tig it's not possible to use the named items. It's only possible when we instantiate a new one. Please review that.

Reply:
I don't understand. If you declare the `DisplaySize` function to return a tuple, and name the items in the tuple (as I showed above), this will all work. Or i'm missing something fundamental. Which could be the case. I swear I've done this before.

Reply:
@tig sorry. Yes you are right. I did something wrong. I'll change. Thanks.